deploy/jenkins-ci/Makefile.helm.jenkins

```diff
@@ -0,0 +1,160 @@
+# version string and the value of HELM_VERSION_TO_RELEASE will be invalid.
+ifeq ($(RELEASE_TYPE),patch)
+  # For patch, bump the version that is present in Makefile. This will be the
+  # version string to use for the release.
+  HELM_VERSION_TO_RELEASE ?= v$(shell semver bump $(RELEASE_TYPE) $(HELM_VERSION))
+endif
+ifeq ($(findstring $(RELEASE_TYPE),major minor),$(RELEASE_TYPE))
+  # For major and minor, remove the -SNAPSHOT suffix from the Makefile
+  HELM_VERSION_TO_RELEASE ?= v$(shell semver bump release $(HELM_VERSION))
+endif
+
+ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else ifeq ($(RELEASE_TYPE),edge)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = n
+endif
+
+ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+  # For end-of-week snapshot release, substitute the -SNAPSHOT prefix in the Makefile with
+  # the value of $(RELEASE_TYPE) -- assuming value will be snapshot.{X}
+  HELM_VERSION_TO_RELEASE ?= v$(shell semver bump prerel $(RELEASE_TYPE) $(HELM_VERSION))
+else ifeq ($(RELEASE_TYPE),edge)
+  # Use 'latest' for edge releases
+  HELM_VERSION_TO_RELEASE ?= latest
+endif
+
+ifeq ($(RELEASE_TYPE),edge)
+  # Avoid semver error, as HELM_VERSION will return empty string when using 'latest' as the version
+  # By not setting these variable, we avoid a semver error when expanding a rule that uses $(HELM_BUMPED_VERSION)
+else
+  # NOTE: The value fo HELM_BUMPED_VERSION and HELM_VERSION_BRANCH will only be valid
+  # after the main Makefile has been modified with the version being released.
+  HELM_BUMPED_VERSION ?= $(shell semver bump minor $(HELM_VERSION))
+  HELM_VERSION_BRANCH ?= $(shell semver bump release $(HELM_VERSION) | sed 's/\.[[:digit:]]\+$$//')
+endif
+
+HELM_GITHUB_URI ?= git@github.com:kiali/helm-charts.git
+HELM_PULL_URI ?= https://api.github.com/repos/kiali/helm-charts/pulls
+HELM_RELEASE_URI = $(subst pulls,releases,$(HELM_PULL_URI))
+HELM_MAIN_BRANCH ?= master
+
+KIALI_BOT_USER ?= kiali-bot
+HELM_FORK_URI ?= $(shell git config --get remote.origin.url)
+
+# BUILD_TAG is an environment variable from Jenkins
+BUILD_TAG ?= prepare-next-version
+BUMP_BRANCH_ID ?= $(BUILD_TAG)
+
+ifeq ($(RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE),y)
+  IS_SNAPSHOT ?= y
+  OMIT_VERSION_BRANCH ?= y
+else
+  IS_SNAPSHOT ?= n
+  OMIT_VERSION_BRANCH ?= y
+endif
+
+# Block edge releases on github
+ifneq ($(RELEASE_TYPE),edge)
+  PUSH_GITHUB_TAG = y
+endif
+
+.PHONY: helm-build-release
+.PHONY: helm-push-version-tag helm-prepare-next-version
+.PHONY: helm-prepare-next-patch-version helm-prepare-master-next-version
+.PHONY: all release
+
+all:
+	$(error You must explicitly specify a target)
+
+helm-build-release:
+	@echo "Will build version: $(HELM_VERSION_TO_RELEASE)"
+	sed -i -r 's/^VERSION \?= v.*/VERSION \?= $(HELM_VERSION_TO_RELEASE)/' Makefile
+	$(MAKE) clean update-helm-repos
+
+helm-push-version-tag:
+ifeq ($(PUSH_GITHUB_TAG),y)
+	git add Makefile
+	git add docs
```

Comment:
This entire Makefile is essentially a direct copy of `Makefile.operator.jenkins` with the exception of:

1. The two lines here at 90 and 95 (where it calls the helm-charts make target `update-helm-repos` and it commits the `docs` folder which is where the helm artifacts are copied so GitHub Pages can find it)
2. Line 71 (`OMIT_VERSION_BRANCH` will always be `y` now - we never want to release into version branches - everything has to be `master` because that is where GitHub Pages gets its content from).
3. Anything dealing with pushing to quay.io has been removed - there are no Helm artifacts getting pushed into quay.

The rest of this file is essentially identical with the obvious exceptions of renaming variables so they say things with "HELM" rather than "OPERATOR".